modules/store/src/selector.ts

```diff
@@ -276,8 +504,19 @@ export function createSelectorFactory(
       return projector.apply(null, selectors);
     });
 
-    const memoizedState = defaultMemoize(function(state: any) {
-      return options.stateFn.apply(null, [state, selectors, memoizedProjector]);
+    const memoizedState = defaultMemoize(function(state: any, props: any) {
+      // createSelector works directly on state
```

Comment:
Is this related or a different feature? Feels like this should be a separate PR

Reply:
I thought you meant this with your comment "I think only having props for selector functions is reasonable."
But you probably meant:
```
createSelector(
   getX,
   (props) => props...
)
```


Is this possible to do? Or do shouldn't we pass the state to the 'props selectors'?

Reply:
What I meant was we don't need to try to support using string selectors with props like below.

```
select('something', 5);
```

Reply:
Ah gotcha 😅 ! 
Do you think having the option to create a selector solely based on props would be worth it?
```
createSelector((state, props) => ...)
```

If so I can pull out [the commit](https://github.com/ngrx/platform/pull/1194/commits/4605ff965dbfd05230d25033bbc7ae96b140663c) out of this PR and create a new one, or I could clean up these commits into 2 commits.

Reply:
2 commits is fine